libs/wire-api/src/Wire/API/User/Search.hs

```diff
@@ -166,6 +167,26 @@ instance ToSchema Contact where
 --------------------------------------------------------------------------------
 -- TeamContact
 
+data Sso = Sso
```

Comment:
```suggestion
-- | Related to `UserSSOId`, but more straight-forward because it does not take SCIM externalId into account.
data Sso = Sso
```

... or something to that end?